packages/common/src/api/comments.ts

```diff
@@ -18,12 +18,13 @@ const optimisticUpdateCommentList = (
   updateRecipe: (prevState: Comment[] | undefined) => void, // Could also return Comment[] but its easier to modify the prevState proxy array directly
   dispatch: ThunkDispatch<any, any, any>,
   userId?: number,
-  page: number = 0
+  page: number = 0,
+  sortMethod: TrackCommentsSortMethodEnum = TrackCommentsSortMethodEnum.Top
 ) => {
   dispatch(
     commentsApi.util.updateQueryData(
       'getCommentsByTrackId',
-      { entityId, userId, limit: 5, offset: page },
+      { entityId, userId, limit: 5, offset: page, sortMethod },
```

Comment:
Note: This was missing after our last sort update. With this addition our optimistic update system is even more brittle than before so once again I'm tempted to refactor our store 💀

Reply:
💀

Reply:
😅 
I did also observe some bugs with sort data so thats a next week task for me to dig into lol